extensions/modules/persistentlogin/src/main/java/org/exist/xquery/modules/persistentlogin/PersistentLogin.java

```diff
@@ -31,6 +31,7 @@
 
 import java.security.SecureRandom;
 import java.util.*;
```

Comment:
Should this be replaced by a list of package imports from java.util?

Reply:
yeas please